src/components/Statistics/Counter/ReportInfo/ReportInfo.js

```diff
@@ -129,20 +129,50 @@ class ReportInfo extends React.Component {
         <FormattedMessage id="ui-erm-usage.general.manualChanges.infoText" />
         <br />
         <FormattedMessage id="ui-erm-usage.general.editReason" />
-        {_.get(this.props.report, 'editReason', '-')}
+        {get(this.props.report, 'editReason', '-')}
       </>
     );
   }
 
+  isWarningCode = code => {
```

Comment:
Is it possible to extract this function to e.g. `src/util`?
Currently, it is used here and in `UDPFilters.js`. If you move it to a separate file you can import it in both, and avoid the redundant definition.

Reply:
Thanks a lot! I did extract that function.